pkg/scheduler/util/utils.go

```diff
@@ -67,13 +67,15 @@ func GetEarliestPodStartTime(victims *extenderv1.Victims) *metav1.Time {
 	maxPriority := corev1helpers.PodPriority(victims.Pods[0])
 
 	for _, pod := range victims.Pods {
-		if corev1helpers.PodPriority(pod) == maxPriority {
-			if GetPodStartTime(pod).Before(earliestPodStartTime) {
-				earliestPodStartTime = GetPodStartTime(pod)
+		earliestPodStartTimeTmp := GetPodStartTime(pod)
+		maxPriorityTmp := corev1helpers.PodPriority(pod)
```

Comment:
please remove Tmp at the end on both.

Reply:
Nah, ok then that'll result in the conflict with existing vars. Sorry, nvm

Reply:
https://github.com/kubernetes/kubernetes/pull/117390 does it in a better way, lets merge that one.